app/assets/stylesheets/addtocalendar.scss

```diff
@@ -107,3 +107,5 @@
 .atc-style-blue .atcb-item:hover {
     background: $text-blue;
 }
+
+.atcb-item-link { font-family: $body-font !important; }
```

Comment:
Actually, looking at the code now I can just change `line 92` to use `$body-font` instead of Verdana. That'd solve both things you mentioned.